.github/workflows/webserver-deploy.yml

```diff
@@ -0,0 +1,49 @@
+name: Build and deploy webserver
+
+on:
+  push:
+    paths:
+      - 'webserver/**'
+      - 'build/webserver/**'
+      - '.github/**'
```

Comment:
The deploy step. We only want to trigger it upon changes in the `webserver` folder, a change in the github actions or the docker config.

:warning:  This was not tested yet :warning: